src/devices/machine/mc68681.cpp

```diff
@@ -669,6 +781,46 @@ void mc68681_device::write(offs_t offset, uint8_t data)
 		duart_base_device::write(offset, data);
 }
 
+void mcf5206e_uart_device::write(offs_t offset, uint8_t data)
+{
+	// The ColdFire has the registers aligned to 32-bits address values, despite being 8-bit registers
+	// Also has a slightly different mapping
+
+	offset /= 4;
+	offset &= 0x0f;
+	switch (offset)
+	{
+	case 0x00: /* UMR */
+	case 0x01: /* UCSR */
+	case 0x02: /* UCR */
+	case 0x03: /* UTB */
+	case 0x04: /* ACR */	// <-- This needs changing as ACR on ColdFire only handles interrupt enable for CTS
```

Comment:
For stuff like this we like to standardize it as "TODO" so you can easily grep the source for things that need to be done.